### PR TITLE
🔌 Add card, grid, tabs directive plugins

### PR DESCRIPTION
**Pull request by fwkoch**
image

This consumes the new packages that were pulled out of `myst-cli` here: https://github.com/executablebooks/mystjs/pull/179 - enables card, grid, and tabs directives.

please update playwright snapshots

### CODE REVIEW

File: src/myst.ts [hunk 1]
```diff
@@ -18,13 +18,22 @@ import { unified } from 'unified';
...
 
 export function markdownParse(text: string): Root {
-  const myst = new MyST();
+  const myst = new MyST({
+    directives: {
+      ...cardDirectives,
+      ...gridDirectives,
+      ...tabsDirectives
+    }
+  });
```
Comment (rowanc1) [inline src/myst.ts:36]: So easy!

@agoose77 -- I think this is a think for a future settings panel. :)

**Review by rowanc1 — APPROVED** [review]
Looks super easy! Thanks @fwkoch -- now to do it for real underneath. :)

**Comment by fwkoch** [comment]
Issue to track doing it for real: https://github.com/executablebooks/mystjs/issues/181

**Comment by rowanc1** [comment]
please update playwright snapshots

**Pull request merged**